site/content/blog/2021-08-01-whats-new-in-svelte-august-2021.md

```diff
@@ -0,0 +1,64 @@
+---
+title: What's new in Svelte: August 2021
+description: Shadow DOM, export and await - oh my!
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+From The Changelog ([JS Party Ep. 182](https://changelog.com/jsparty/182)) to Svelte Radio (Episodes [29](https://share.transistor.fm/s/adc23e84) and [30](https://share.transistor.fm/s/6316622d)), it seems that folks couldn't help but talk about Svelte, this month! Also, shadow DOM support and new export and await functionality are new in Svelte.
+
+## New in Svelte
+- You can now specify a `ShadowRoot` as the `target` when creating a component - making it possible to render Svelte components inside the shadow DOM (**3.40.0**)
+- The `export { ... } from`, `export let { ... } =` and `{#await ... then/catch}` syntaxes are all now supported in Svelte components (**3.41.0**)
+
+For a full list of features and bug fixes, check out the [Svelte changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## SvelteKit Updates
+- More powerful and configurable rendering options are now available ([#2008](https://github.com/sveltejs/kit/pull/2008))
```

Comment:
rich has suggested we not keep this one https://github.com/sveltejs/kit/pull/2047. I haven't gotten a chance to review what he's said (hope to do so today), but we might want to hold off on publicizing

Reply:
cool! Removed for now :)